nservicebus/upgrades/release-policy.md

```diff
@@ -49,7 +49,7 @@ It is strongly recommended to upgrade frequently enough to stay on a supported v
 
 ### Deprecation
 
-Public API methods and properties are deprecated using the [ObsoleteAttribute messages](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx). Continued usage of a deprecated method or property will result either in a compiler warning or error. A warning message indicates the version in which the deprecation will start to generate a compiler error (usually the next major version). The message will also indicate the version in which the deprecated method or property will be removed. Deprecations in minor releases will always generate a compiler warning and not an error.
+Public API members are deprecated using [ObsoleteAttribute](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx) messages. Continued usage of a deprecated member will result in either a compiler warning or error. A warning message indicates the version in which the deprecation will start to generate a compiler error (usually the next major version). Both warning and compiler messages indicate in which version the deprecated member will be removed. Deprecations in minor releases will always generate a compiler warning and not an error.
```

Comment:
@adamralph - I dislike members. API members are not commonly known are used. People use public interfaces, classes methods and properties. Although members might comprise of delegates, events etc, it is irrelevant. Are our consumers using that? What are the most common public facing types - it is most likely interfaces, classes, methods and properties. So be explicit.